ui/app/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "regula",
+  "version": "0.6.0",
```

Comment:
❓ Why do we are sync between the backend version and the UI one?

Reply:
It should be `0.1.0`, isn't it?

Reply:
It depends, they are part of the same project so I think that we want both projects to be synchronized, at least for now